### PR TITLE
Multi-arch build

### PR DESCRIPTION
**Pull request by jari-hodju**
- re-branched from main due to long list of merge conflicts
- updates / changes derived from git diff main..feat-multiarch
- sdk and base image upgrades
- github actions upgrades
- for general info on multi-arch builds, see https://github.com/tiiuae/fog-ros-baseimage/blob/main/docs/Multiarchitecture_builds.md

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -9,7 +9,3 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
```
Comment (mehmetkillioglu) [inline .github/dependabot.yml:11]: Why the github actions were removed from the dependabot checks?

Also, for docker, we would need to specify like this so it groups the baseimage updates in one PR

```
- package-ecosystem: "docker"
  directory: "/"
  schedule:
    interval: "daily"
  groups:
     dev-dependencies:
        patterns:
          - "tiiuae/fog-ros-sdk"
          - "tiiuae/fog-ros-baseimage-builder"
          - "tiiuae/fog-ros-baseimage"
 ```

**Review by mehmetkillioglu — CHANGES_REQUESTED** [review]
The rebasing did not succeed unfortunately. This PR makes it so that it tries to remove already existing changes to the main branch. It removes the coverity workflow, and changes from 
#26, #27, and #35. The only required changes are tii-mocap-pose.yaml, Dockerfile, and entrypoint.sh. All the other content should be exactly the same.

File: .github/dependabot.yml (hunk 1 above)
Comment (jari-hodju) [reply .github/dependabot.yml]: Fixed with e295e736bad44f42265b49e88015f4dd1888d8e4

Comment (jari-hodju) [reply .github/dependabot.yml]: Put back the githut action dependabot definition with eecc410f22fe418408ab0b9a8fd9a525ab7de8ee

**Pull request closed without merging**